setup.py

```diff
@@ -68,7 +68,7 @@
     py_modules=["stagedfright"],
     install_requires=[
         "pyomo>=6.2",
-        "pandas==1.2.*",
+        "pandas ~= 2",
```

Comment:
@lbianchi-lbl is `~= 2`  valid syntax here? This seems to be causing all of the tests to fail.

Reply:
Ah, excellent point, thanks. I thought it would work but clearly it doesn't. I've just changed it to `>=` since I don't think it matters much.